roxie/roxiemem/roxiemem.cpp

```diff
@@ -8243,7 +8240,7 @@ class RoxieMemTests : public CppUnit::TestFixture
         }
         void * otherrow = rowManager->allocate(100, 0);
         savedRowHeap.setown(rowManager->createFixedRowHeap(8, ACTIVITY_FLAG_ISREGISTERED|0, RHFhasdestructor|RHFunique));
-        void * leakedRow = savedRowHeap->allocate();
+        [[maybe_unused]] void * leakedRow = savedRowHeap->allocate();
```

Comment:
Would the following be better?

void * leakedRow [[maybe_unused]] = savedRowHeap->allocate();

Reply:
Maybe, slightly. I assume they behave the same?